src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 springdoc.swagger-ui.path=/swagger-ui
+admin_password = ENC(LSzhwAqrhqwuOT9tIB/gb2bn2ezT8h8G5Rj055kTyMnycNKnpN/C/0gubMJE3lWh)
+user_password = ENC(oCv5ZsKW2NLDnwPSdevVqWzcwE3ifDv0MFc8WTPuz5szYJ0DmpTeP4j/4bvAfh5q)
+jasypt.encryptor.password=6jURGwGr6dBA
```

Comment:
shouldn't it be store in envs? Can I use this values to encrypt the data myself if I know them?

Reply:
is it ok now? i added env variables to the render

Reply:
i set default value of env variables for tests and CI, because they don't have to have real values

Reply:
Now tests ignore getting value of env variables